src/Components/Samples/BlazorServerApp/Data/WeatherForecast.cs

```diff
@@ -10,6 +10,6 @@ public class WeatherForecast
 
         public int TemperatureF => 32 + (int)(TemperatureC / 0.5556);
 
-        public string Summary { get; set; }
+        public string Summary { get; set; } = default!;
```

Comment:
Why not make it a nullable string (`string?`)?

Reply:
In this particular case, Summary is always initialized, so this better reflects the nullability of the property.

Reply:
@pranavkm If I do `var forecast = new WeatherForecast();`, it won't be initialized. Giving it a default value of `string.Empty` could be better IMO.